src/maliput_malidrive/builder/road_curve_factory.h

```diff
@@ -345,6 +353,12 @@ class RoadCurveFactory final : public RoadCurveFactoryBase {
   std::unique_ptr<malidrive::road_curve::Function> MakeCubicFromXodr(
       const std::vector<T>& xodr_data, double p0, double p1, FillingGapPolicy policy,
       road_curve::PiecewiseFunction::ContinuityCheck continuity_check) const;
+
+  // Constructs CubicPolynomials out of @p lane_widths.
+  // TODO(@Santoi): Add docstring.
+  std::vector<std::unique_ptr<malidrive::road_curve::Function>> MakeCubicPolynomialsLaneWidths(
```

Comment:
nit in the name: MakeCubicPolynomialsFromLaneWidths

Reply:
Done.